app/controllers/v1/users_controller.rb

```diff
@@ -21,7 +21,10 @@ def auth
       user.generate_login_code!
 
       if user.save
-        ApplicantMailer.login_code(user).deliver_later
+        if user.sms? && TwilioClient.send_login_code(user)
+        else
+          ApplicantMailer.login_code(user).deliver_later
+        end
 
         render_success(
           id: user.id,
```

Comment:
Response should differentiate with what kind of login code was sent (SMS or email).

Reply:
So we can display on the frontend where the login code was sent to